seg_lapa/train.py

```diff
@@ -11,14 +11,16 @@
 from seg_lapa.config_parse import train_conf
 from seg_lapa.config_parse.train_conf import TrainConf
 from seg_lapa.loss_func import CrossEntropy2D
+from seg_lapa.utils.path_check import get_project_root
 
 
 class DeeplabV3plus(pl.LightningModule):
-    def __init__(self, config: TrainConf):
+    def __init__(self, config: TrainConf, run_id: str):
         super().__init__()
         self.cross_entropy_loss = CrossEntropy2D(loss_per_image=True, ignore_index=255)
         self.config = config
         self.model = self.config.model.get_model()
+        self.logs_dir = get_project_root() / 'logs' / run_id
```

Comment:
1. If the directory does not already exist, we need to create it. 
2. Any constants such as `'logs'` should be saved as a constant somewhere (top of file or init of class)

Let's create a function to create logs dir and directly pass the path to logs dir to this class

Reply:
1. Lightning takes care about creating directory if it does not exist

Reply:
Huh? Could you please link to the docs page or code sample. 
This is literally an attribute we're adding to the lightning module, how does it create the dir?

Reply:
@GainiK Do you mean by setting `self.log_dir` in LightningModule, pl will create the dir for us?

Reply:
(this is different from the `self.logs_dir` above.)